src/NovelRT/DebugService.cpp

```diff
@@ -30,7 +30,7 @@ namespace NovelRT {
     _runner(runner),
     _fpsCounter(nullptr),
     _framesPerSecond(0) {
-    runner->subscribeToSceneConstructionRequested(std::bind(&DebugService::onSceneConstruction, this));
+    runner->SceneConstructionRequested += std::bind(&DebugService::onSceneConstruction, this);
```

Comment:
IIRC, there was some kind of lambda replacement for `std::bind` - though I can't say I quite understand what that replacement would be or look like.

Your call if you wanna investigate this further.

Reply:
Not in this PR. If you want to log an issue, we can follow up in the future.